cmd/apiserver-proxy-sidecar/Dockerfile

```diff
@@ -1,20 +1,21 @@
 #############      builder              #############
-FROM golang:1.13.7 AS builder
+FROM l.gcr.io/google/bazel:2.1.0 as builder
```

Comment:
This image is almost double the size of the previous image, also how did this happen (creation time of 50 years) happen :D 

```
l.gcr.io/google/bazel    2.1.0     6b22f20532d8       50 years ago        1.6GB                                                                                              
```

Reply:
All images created by bazel have their timestamps set to epoch time for reproducibility. 

I would hope to get [remote execution](https://docs.bazel.build/versions/2.1.0/remote-execution.html) and [remote caching](https://docs.bazel.build/versions/2.1.0/remote-caching.html) in our CI/CD eventually and it would speed the build time considerably.